src/util/accessibility/live.ts

```diff
@@ -0,0 +1,49 @@
+import {Injectable, Inject, InjectionToken, OnDestroy} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+
+
+// usefulness (and default value) of delay documented in Material's CDK
+// https://github.com/angular/material2/blob/6405da9b8e8532a7e5c854c920ee1815c275d734/src/cdk/a11y/live-announcer/live-announcer.ts#L50
+export type DELAY_TYPE = number | null;
```

Comment:
Could we name them more precisely, please? `{provide: DELAY, ...}` - not clear which delay.

Maybe something like `ARIA_LIVE_DELAY` ?